libraries-2/pom.xml

```diff
@@ -16,15 +16,7 @@
         <repository>
             <id>jboss-public-repository-group</id>
             <name>JBoss Public Repository Group</name>
-            <url>http://repository.jboss.org/nexus/content/groups/public/</url>
-            <releases>
-                <enabled>true</enabled>
-                <updatePolicy>never</updatePolicy>
-            </releases>
-            <snapshots>
-                <enabled>true</enabled>
-                <updatePolicy>daily</updatePolicy>
-            </snapshots>
+            <url>https://repo.maven.apache.org/maven2</url>
```

Comment:
do we need to add this repository here explicitly? I think this is the default repository

Reply:
Yes, you are right. I remove the entire block for **<repositories>**.